css-fonts/styles.css

```diff
@@ -0,0 +1,10 @@
+h1, h2{
+  font-family: 'Libre Barcode 39', cursive;
+  padding-top: 20px;
+  color: lightblue;
+}
+
+p {
+  color: lightgoldenrodyellow;
+  line-height: 35px;
+  }
```

Comment:
You imported 2 different fonts, but you're only actually using one. You should have one font applied to your headings, and your second font should be applied to the rest of the content or the `p` tag in this case.